presto-product-tests/conf/presto/etc/config.properties

```diff
@@ -31,6 +31,8 @@ plugin.bundles=\
   ../../../presto-hive-cdh5/pom.xml,\
   ../../../presto-teradata-functions/pom.xml,\
   ../../../presto-tpch/pom.xml,\
+  ../../../presto-postgresql/pom.xml,\
```

Comment:
That's strange... How did the product tests pass without that on master? I re-enabled postgresql and mysql tests here: https://github.com/prestodb/presto/pull/5761